enterprise-suite/templates/alertmanager-configmap.yaml

```diff
@@ -3,6 +3,11 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   labels:
+    app.kubernetes.io/name: {{ template "name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: alertmanager
     app: prometheus
```

Comment:
Other resources no longer have `app:` and `component:` labels. Is there a reason to have them for alertmanager configmap and deployment?

Reply:
Not for the `ConfigMap`, but for the `Deployment` it's necessary to preserve backwards compatibility for upgrades.